cadc-util/src/main/java/ca/nrc/cadc/db/version/KeyValueDAO.java

```diff
@@ -197,6 +198,41 @@ public void delete(String name) {
         jdbc.update(sql, arg);
     }
 
+    public List<KeyValue> list() {
+        String select = String.format("SELECT %s, %s, %s FROM %s", columnNames[0], columnNames[1], columnNames[2], tableName);
```

Comment:
The `select` SQL is almost a duplicate of the sql generated inside the SelectStatementCreator; if you made the WHERE (name) optional, the SQL generation and probably the whole SelectStatementCreator could be re-used.

Reply:
Used the SelectStatementCreator for both get() and list() methods.